model_compression_toolkit/core/common/mixed_precision/resource_utilization_tools/ru_methods.py

```diff
@@ -87,6 +90,88 @@ def weights_size_utilization(mp_cfg: List[int],
     return np.array(weights_memory)
 
 
+def calc_graph_cuts(graph: Graph) -> List[Cut]:
+    """
+    Calculate graph activation cuts.
+    Args:
+        graph: A graph object to calculate activation cuts on.
+
+    Returns:
+        A list of activation cuts.
+
+    """
+    memory_graph = MemoryGraph(deepcopy(graph))  # TODO maxcut: do we need the deepcopy?
+    _, _, cuts = compute_graph_max_cut(memory_graph)
+
+    if cuts is None:
```

Comment:
`
filtered_cuts = None 
if cuts is not None: 
... 

return filtered_cuts
`



I think it's simpler but up to you